static/src/javascripts/projects/common/modules/commercial/hosted-gallery.js

```diff
@@ -299,6 +301,8 @@ define([
                     bonzo(this.$galleryEl).toggleClass('show-oj', this.index === this.$images.length);
                     bonzo(this.$galleryEl).toggleClass('show-cta', this.index === config.page.ctaIndex + 1);
                 }
+
+                url.pushUrl({}, document.title, config.page.pageName + '#img-' + this.index, true);
```

Comment:
Ah, I didn't realise - I assumed by `push` it meant append as it would in array land.
👍  for clarifying.